e2e/terraform/nomad-acls.tf

```diff
@@ -15,21 +15,14 @@ locals {
 resource "null_resource" "bootstrap_nomad_acls" {
   depends_on = [module.nomad_server]
   triggers = {
-    script = data.template_file.bootstrap_nomad_script.rendered
+    script = "${local.nomad_env} ./scripts/bootstrap-nomad.sh"
   }
 
   provisioner "local-exec" {
-    command = data.template_file.bootstrap_nomad_script.rendered
+    command = aws_instance.server.0.public_ip != "" ? local.nomad_env : "echo 'Nomad server not ready yet, skipping bootstrap'"
   }
 }
```

Comment:
I'm not sure what this is for... if there's no public IP, wouldn't we be unable to run the bootstrapping in the first place and therefore just halt when the bootstrap skip fails?